packages/block-editor/src/store/actions.js

```diff
@@ -658,6 +658,305 @@ export const synchronizeTemplate = () => ( { select, dispatch } ) => {
 	dispatch.resetBlocks( updatedBlockList );
 };
 
+function mapRichTextSettings( attributeDefinition ) {
+	const {
+		multiline: multilineTag,
+		__unstableMultilineWrapperTags: multilineWrapperTags,
+		__unstablePreserveWhiteSpace: preserveWhiteSpace,
+	} = attributeDefinition;
+	return {
+		multilineTag,
+		multilineWrapperTags,
+		preserveWhiteSpace,
+	};
+}
+
+/**
+ * Delete the current selection.
+ *
+ * @param {boolean} isForward
+ */
+export const deleteSelection = ( isForward ) => ( {
```

Comment:
I wonder if these two new actions (deleteSelection, splitSelection) deserve their own unit tests. I'm not sure we can test everything here in e2e tests. WDYT